src/XperienceCommunity.Localization.Base/Localizer/XperienceStringLocalizerFactory.cs

```diff
@@ -0,0 +1,31 @@
+﻿using CMS.ContentEngine;
+using CMS.DataEngine;
+using CMS.Helpers;
+using CMS.Websites.Routing;
+using Microsoft.Extensions.Localization;
+
+namespace XperienceCommunity.Localizer.Internal
+{
+    internal class XperienceStringLocalizerFactory(IStringLocalizerFactory baseStringLocalizerFactory,
+        IProgressiveCache progressiveCache,
+        IWebsiteChannelContext websiteChannelContext,
+        IInfoProvider<ContentLanguageInfo> contentLanguageInfoProvider) : IStringLocalizerFactory
+    {
+        private readonly IStringLocalizerFactory baseStringLocalizerFactory = baseStringLocalizerFactory;
+        private readonly IProgressiveCache progressiveCache = progressiveCache;
+        private readonly IWebsiteChannelContext websiteChannelContext = websiteChannelContext;
+        private readonly IInfoProvider<ContentLanguageInfo> contentLanguageInfoProvider = contentLanguageInfoProvider;
```

Comment:
With primary constructor these lines are obsolete, right? Can you remove them?

Reply:
removed since not using internal private readonly property construction format.